toonz/sources/image/pli/pli_io.cpp

```diff
@@ -473,6 +473,8 @@ void ParsedPli::setMaxThickness(double maxThickness)
 	imp->m_maxThickness = maxThickness;
 };
 
+#define ABS(a)  ((a) < 0 ? (-(a)) : (a))
```

Comment:
This was using `std::abs` already.
The issue is passing `char` and `short` gets converted to a `double` by GCC5.3, which then fails to work with bitmasks.

Another alternative is to cast to int.

eg:

    return (UCHAR)(abs((int)val) | (val & 0x80));

Or we could have a local inline function for char and short `abs`, instead of the macro.